libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -306,7 +306,7 @@ RC_Channel::update_min_max()
   the current radio_in value using the specified dead_zone
  */
 int16_t
-RC_Channel::pwm_to_angle_dz_trim(uint16_t dead_zone, uint16_t _trim)
+RC_Channel::pwm_to_angle_dz_trim(uint16_t dead_zone, uint16_t _trim)const
```

Comment:
I'm not sure if there's a specific rule for this, but I think that a space before const would be appreciated